packages/docs/locales/en/en.json

```diff
@@ -1872,5 +1891,35 @@
         "text": "By default a `va-backtop` watches for body element offset height. You can change target element, positioning and scrolling speed."
       }
     }
+  },
+  "toast": {
+    "title": "Toast",
```

Comment:
We want at least several lines on how service is different from component. Maybe in QA below the page.